composer.json

```diff
@@ -15,7 +15,8 @@
     "johnpbloch/wordpress": "5.4.2",
     "altis/cms-installer": "^0.4.2",
     "johnbillion/extended-cpts": "^4.4.0",
-    "humanmade/clean-html": "^2.0.0"
+    "humanmade/clean-html": "^2.0.0",
+    "humanmade/post-cloner": "^1.0.0"
```

Comment:
Let's set this to `~1.0.0` so it just tracks patch versions - I'm starting to update the version pinning for our plugins in this way across the board from now on.

Reply:
actually it will be `1.1.1`